draft-packages/collapsible/KaizenDraft/Collapsible/CollapsibleGroup.tsx

```diff
@@ -1,23 +1,23 @@
+import React from "react"
 import classnames from "classnames"
-import * as React from "react"
-
+import { CollapsibleProps } from "./Collapsible"
 import styles from "./styles.scss"
 
-type Props = {
+export type Sticky = {
+  top: string
+}
+
+type CollapsibleGroupProps = {
+  children: Array<React.ReactElement<CollapsibleProps>>
   separated?: boolean
   sticky?: Sticky
   noSectionPadding?: boolean
   automationId?: string
   lazyLoad?: boolean
   onToggle?: (open: boolean, id: string) => void
-  children: Array<React.ReactElement<any>>
 }
 
-export type Sticky = {
-  top: string
-}
-
-export const CollapsibleGroup: React.FunctionComponent<Props> = ({
+export const CollapsibleGroup: React.VFC<CollapsibleGroupProps> = ({
```

Comment:
I needed to use `VFC` instead of `FC` because that was part of the reason why it broke (`FC` says that `children: React.ReactNode` whereas we did not want that; `VFC` does not come with children)